### DISCUSSION TITLE
No input control for Blackhole

### DISCUSSION THREAD

**Discussion by myershse** [ExistentialAudio/BlackHole#317] [General]
Installed Blackhole on MacBook Air M1 with macOS 11.2.1.  System Preferences reports no input control.
<img width="666" alt="Screen Shot 2021-02-24 at 1 46 31 PM" src="https://user-images.githubusercontent.com/4411827/109057040-dbd8cf80-76a6-11eb-95cc-9cb9c391d4d5.png">

**Comment by devinroth** [comment]
Correct. Just use the output volume.
